.gitignore

```diff
@@ -17,7 +17,7 @@ requirements.txt
 output/
 .DS_Store
 tests/test_storage
-tests/test_workspaces
+tests/test_descriptor_generator
```

Comment:
I renamed the output directory for the tests, I think `test_workspaces` was too general and not really what the directory was (it was more like a package directory, so one level down).